doc/cephfs/fuse.rst

```diff
@@ -2,38 +2,60 @@
 Mount CephFS using FUSE
 =======================
 
-Before mounting Ceph via "Filesystem in USErspace" (FUSE), ensure that the client
-host has a copy of the Ceph configuration file and a keyring with CAPS for the
-Ceph metadata server.
+Pre-requisite
+=============
 
-#. From your client host, copy the Ceph configuration file from the monitor host 
-   to the ``/etc/ceph`` directory. :: 
+Before mounting Ceph via "Filesystem in USErspace" (FUSE), ensure that the
+client host (where CephFS has to mounted and used) has a copy of the Ceph
+configuration file (i.e. ``ceph.conf``) and a keyring of CephX user that has
+CAPS for the Ceph metadata server. Both of these files must be on the host
+where the Ceph monitor resides.
 
+#. Copy the Ceph conf from the monitor host to client host at
+   ``/etc/ceph``::
+
+	# on client host
 	sudo mkdir -p /etc/ceph
 	sudo scp {user}@{server-machine}:/etc/ceph/ceph.conf /etc/ceph/ceph.conf
 
-#. From your client host, copy the Ceph keyring from the monitor host to 
-   to the ``/etc/ceph`` directory. :: 
+#. Ensure that the conf has appropriate permissions::
+
+	chmod 644 /mnt/cephfs
+
+#. Copy the Ceph keyring from the monitor host to client host at
+   ``/etc/ceph``::
```

Comment:
Better to recommend they create a key for this host with a command like
```
ceph auth get-or-create client.fs.$hostname mon ...
```
I'm not sure what the minimal/recommended caps are for fs clients these days... @batrick?

Reply:
We recommend `ceph fs authorize` now to make it easy and automatic.

Reply:
Also, please make the same change to kernel.rst.

Reply:
Yep. I've added the command to both the pages and tested it myself to be sure.